container_images/validate-integtest/validate-result.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -x
+
+echo "Validate Integration Test Result"
+
+gcloud auth activate-service-account --key-file="$GOOGLE_APPLICATION_CREDENTIALS"
+gsutil cp "$GCS_PATH"/go-test*.txt ./
+
+RET=0
+
+# Convert txt report to xml and html
+for f in go-test*.txt; do
+  if [ "`grep -c "FAIL" "$f"`" -gt "0" ]; then
```

Comment:
if you just want to check exit code of grep, try like this:
```
if ! grep -qc "FAIL" "$f"; then
  RET=1
```
and set ret rather than using `$?`, because this will be the result of the [ statement anyway

Reply:
I think it should be ```if grep -qc "FAIL" "$f"; then```